src/py21cmfast/src/PerturbHaloField.c

```diff
@@ -58,6 +58,8 @@ int ComputePerturbHaloField(float redshift, InitialConditions *boxes, HaloField
         growth_factor = dicke(redshift);  // normalized to 1 at z=0
         displacement_factor_2LPT = -(3.0 / 7.0) * growth_factor * growth_factor;  // 2LPT eq. D8
 
+        // TODO: combine/match with PerturbField.c
+        //  which uses (D(z) - D(init))/BOXLEN
```

Comment:
Marking this as something to do.

Reply:
I would classify this under #495. Since ADACS did a bit of this reorganisation in their work I'm planning to piggyback off their efforts in the next big PR to make everything uniform there.

Reply:
Cool, no worries